website/src/css/customTheme.scss

```diff
@@ -399,24 +399,24 @@ html[data-theme="dark"] article .badge {
 
 /* Admonitions & notes */
 
-.alert--warning {
-  background-color: var(--rn-note-background);
-  font-size: 14px;
-  text-align: center;
+.admonition,
+.theme-admonition {
+  font-size: var(--ifm-blockquote-font-size);
+  margin-bottom: 1.5rem !important;
 
-  .margin-top--md {
-    margin-top: 0.33rem !important;
+  div[class^="admonitionContent"] {
+    h5 {
+      font-size: 14px;
+    }
```

Comment:
As far as I can tell, this `h5` rule is unused. Let's remove it :)

Reply:
Good catch, they changed the `h5` to `div` with `admonitionHeader` in the class name, this should target the admonition title.